test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-provisioner.yaml

```diff
@@ -40,7 +40,8 @@ spec:
       serviceAccountName: csi-provisioner
       containers:
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.4.0
+          # TODO: replace with official 1.5.0 release when ready
+          image: quay.io/k8scsi/csi-provisioner:v1.5.0-rc1
```

Comment:
Does provisioner rbac need to be updated to read the new beta snapshot objects?

Reply:
No changes are needed.  It's the same API group and resource names.